mlflow/models/evaluation/base.py

```diff
@@ -751,6 +768,8 @@ def evaluate(
        support multi-class classifier, in this case, default evaluator will fallback to use
        shap Exact or Permutation explainer.
```

Comment:
Can we change lines 766 -  769 to:

```
     - The `shap.Explainer` "auto" algorithm will use the Linear explainer for linear models
       and the Tree explainer for tree models. Because SHAP's Linear and Tree explainers
       do not support multi-class classification, the default evaluator will fall back to using
       the Exact or Permutation explainers for multi-class classification tasks.
```